core-web/libs/dojo-theme/scss/backend/dot-admin/forms/_base.scss

```diff
@@ -33,13 +33,16 @@
 .required {
     &:before {
         @extend #dijit-icon;
-        content: $fa-var-asterisk;
-        color: $error;
         font-size: 9px;
         position: relative;
         top: -4px;
         left: 2px;
     }
+
+    &::after {
+        content: $fa-var-asterisk;
```

Comment:
<img width="885" alt="image" src="https://user-images.githubusercontent.com/751424/214058649-466b776b-6396-4b7d-99ed-a805d97d7fff.png">

This is for the font-awesome icons, can be used here, in this case is just `*`

Reply:
Also this needs some padding-left, 2px

Reply:
updated